3.0/README.md

```diff
@@ -0,0 +1,171 @@
+
+```JSON
+{
+  "bounds": [ -180, -85.05112877980659, 180, 85.0511287798066 ]
+}
+```
+
+## 3.3 `center`
+## 3.4 `data`
+## 3.5 `description`
+
+OPTIONAL. Default: null.
+
+A text description of the tileset. The description can contain any legal character. Implementations SHOULD NOT interpret the description as HTML.
+
+```JSON
+{
+  "description": "We are volcanoes. When we women offer our experience as our truth, as human truth, all the maps change. There are new mountains."
+}
+```
+
+## 3.6 `grids`
+## 3.7 `legend`
+
+## 3.8 `maxzoom`
+
+OPTIONAL. Default: 30. >= 0, <= 30.
+
+An integer specifying the maximum zoom level. MUST be >= minzoom.
+
+```JSON
+{
+  "maxzoom": "11"
+}
+```
+
+## 3.9 `minzoom`
+
+OPTIONAL. Default: 0. >= 0, <= 30.
+
+An integer specifying the minimum zoom level. MUST be <= maxzoom
+
+```JSON
+{
+  "minzoom": "0"
+}
+```
+
+## 3.10 `name`
+
+OPTIONAL. Default: null.
+
+A name describing the tileset. The name can contain any legal character. Implementations SHOULD NOT interpret the name as HTML.
+
+```JSON
+{
+  "name": "Earthsea v2"
+}
+```
+
+## 3.11 `scheme`
+
+OPTIONAL. Default: "xyz". 
+
+Either "xyz" or "tms". Influences the y direction of the tile coordinates. The global-mercator (aka Spherical Mercator) profile is assumed.
+
+```JSON
+{
+  "scheme": "xyz"
+}
+```
+
+## 3.12 `template`
+## 3.13 `tilejson`
+
+REQUIRED.
+
+A semver.org style version number. Describes the version of the TileJSON spec that is implemented by this JSON object.
+
+```JSON
+{
+  "tilejson": "3.0"
+}
+```
+
+## 3.14 `tiles`
+
+REQUIRED. 
+
+An array of tile endpoints. {z}, {x} and {y}, if present, are replaced with the corresponding integers. If multiple endpoints are specified, clients may use any combination of endpoints. All endpoints MUST return the same content for the same URL. The array MUST contain at least one endpoint. The tile extension is NOT limited to any particular format. Some of the more popular are: mvt, vector.pbf, png, webp, and jpg.
+
+```JSON
+{
+  "tiles": [
+    "http://localhost:8888/admin/1.0.0/world-light,broadband/{z}/{x}/{y}.mvt"
+  ]
+}
+```
+
+## 3.15 `vector_layers`
+## 3.16 `version`
+
+OPTIONAL. Default: "1.0.0".
+
+A [semver.org](https://semver.org) style version number of the tiles. When changes across tiles are introduced the minor version MUST change. This may lead to cut off labels. Therefore, implementors can decide to clean their cache when the minor version changes. Changes to the patch level MUST only have changes to tiles that are contained within one tile. When tiles change significantly, such as updating a vector tile layer name, the major version MUST be increased. Implementations MUST NOT use tiles with different major versions.
```

Comment:
Would the right approach for a version for continually updating data to be to specify build metadata like semver.org 2.0.0 states, e.g. `1.0.0+20130313144700`?